### PR TITLE
Map, legend, and label colors (V1)

### LINKED ISSUE #118: Updated map colors and legend
Need signoff on proposal from Sonya.

### PR DESCRIPTION
**Pull request by sarahgreen**
This is likely to need updating once FreeFrom has done a final pass at design, but in the meantime, this is a more polished version of the colors to use for the map, legend, and score labels.

aka WIP for #118

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -46,7 +47,7 @@ function Home({ states }) {
                 <div className={`col-12 mb-4${mapClass}`}>
                     <div className='float-right'>
                         <div><small>Survivor wealth friendliness</small></div>
-                        <div id='legend' style={{height: '40px', width: '200px', backgroundColor: 'blue'}}/>
+                        <Image src="/images/legend.png" height="40" width="200"/>
```
Comment (egiurleo) [inline pages/index.js:50]: I think this looks great but I'm not sure how accessible it is. I'm wondering whether users with screen readers will need a legend (e.g. will the ranking of each state just be read out for them based on the map data?). Also wondering if this should have an `alt` tag. @DanielKim1 probably knows more than me!

Comment (DanielKim1) [reply pages/index.js:50]: Good catch Emily! Looks like NextJS does _not_ automatically handle accessibility for us.
I'm thinking that the legend is useless for screen reader users anyway:
Screen Shot 2021-03-10 at 9 17 11 AM
So I think we should just hide the whole thing from screen readers by adding the `aria-hidden="true"` attribute to `div.float-right`.

Comment (sarahgreen) [reply pages/index.js:50]: Thanks! @DanielKim1 could you take a quick look at my latest commit to make sure I added the attribute correctly?

Comment (DanielKim1) [reply pages/index.js:50]: looks perfect!

**Pull request merged**